ci/scripts/check-generated.sh

```diff
@@ -32,3 +32,11 @@ if [[ $? != 0 ]]; then
   echo "Autogenerated LC state not up-to-date. Regenerate with 'make -C hw lc-state-enc'."
   exit 1
 fi
+
+hw/ip/flash_ctrl/util/flash_ctrl_gen.py && git diff --exit-code
+if [[ $? != 0 ]]; then
```

Comment:
thanks @rswarbrick . 
This is part of the pain with the templated modules... their local and top counterpart get out of sync.
I'll add an option to the hw/Makefile to also generate the local one also.